cmd/infra/aws/destroy.go

```diff
@@ -78,8 +81,17 @@ func NewDestroyCommand() *cobra.Command {
 }
 
 func (o *DestroyInfraOptions) Run(ctx context.Context) error {
+	if o.AwsInfraGracePeriod.Seconds() == 0 {
```

Comment:
does this mean there's no way to set indefinite timeout? can we please keep current behaviour and let this be opt-int?

Reply:
addressed with [commit](https://github.com/openshift/hypershift/pull/2967/commits/4a34b10409ac327d80bb1cc62c28e2f224bcac1f)